build.sbt

```diff
@@ -227,7 +232,52 @@ def writeStableVersion(selectVersion: Versions => String): ReleaseStep = { st: S
   st
 }
 
+// Commit stable version
```

Comment:
Why this change?

Reply:
This simplifies the release so that `stable-version.txt` is not uncommitted before we push the new version. I don't really mind if this sticks around here, but it fits into the `sbt-release` pattern of modifying files with a version and immediately committing.

Reply:
Can we 1) make this a function to commit a set of files 2) move it to separate file? I don't really like having functions inline with the build definition.